about-me.html

```diff
@@ -0,0 +1,56 @@
+<html>
+<head>
+	<title>Saskia West Software Dev Portfolio</title>
+	<link href="normalize.css" rel="stylesheet" type="text/css">
+	<link href="style.css" rel="stylesheet" type="text/css">
+	<link href="https://fonts.googleapis.com/css2?family=Cinzel&family=Girassol&family=Gloria+Hallelujah&family=Montserrat:wght@200&family=Open+Sans+Condensed:wght@300&display=swap" rel="stylesheet">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+
+<body>
+	
+<section class="top-bar">
+	
+	<nav class="sw-logo">S W</nav>
+	
+	<nav class="nav-bar">
+		<a href="index.html">Home</a>
+		<a href="about-me.html">About Me</a>
+		<a href="portfolio.html">Portfolio</a>
+		<a href="contact.html">Contact</a>
+	</nav>
+	
+	<button class="hamburger">Menu</button>
+	
+</section>
+	
+<main class="about-main">
+	<div><h1>About Me</h1></div>
+	
+	<div class="about-container">
+	
+		<img class="saskia-image" src="saskiawestbw.jpeg" alt="picture of Saskia">
+	
+	<p class="about-p">Theologian turned actor turned coder...(interesting combination?) all whilst navigating the Cornish coast, wild swimming, and making macarons. <br><br> Last year I was studying Hindu feminism for my finals. This year I'm still studying - and still ferociously typing away in front of a screen -  just this time it's in an entirely different language which is rather exciting (albeit somewhat alien at times). I've always been one for variety in life, studying, and never fitting into boxes - quite literally.<br><br>I graduated in 2020 from St John's College, Cambridge University where I studied Theology. Between translating Greek scripture and discussing female empowerment within Hindu communities you'd find me on the stage or walking the occasional fashion show (no tumbles to date...). (To be continued)</p>
```

Comment:
this paragrapgh might not always be a paragraph, so maybe something other than p (in the class)